earn/src/components/lend/BorrowingWidget.tsx

```diff
@@ -167,216 +168,232 @@ export default function BorrowingWidget(props: BorrowingWidgetProps) {
-                  const fact = 1 + 1 / ALOE_II_MAX_LEVERAGE + 1 / ALOE_II_LIQUIDATION_INCENTIVE;
-                  let ltv = 1 / (Math.exp((account.nSigma * account.iv) / 10) * fact);
-                  ltv = Math.max(0.1, Math.min(ltv, 0.9)) * 100;
-                  const token0Color = tokenColors.get(account.token0.address);
-                  const token0Gradient = token0Color
-                    ? `linear-gradient(90deg, ${rgba(token0Color, 0.25)} 0%, ${GREY_700} 100%)`
-                    : undefined;
-                  const token1Color = tokenColors.get(account.token1.address);
-                  const token1Gradient = token1Color
-                    ? `linear-gradient(90deg, ${rgba(token1Color, 0.25)} 0%, ${GREY_700} 100%)`
-                    : undefined;
+    <>
+      <div className='flex gap-4'>
+        <CardWrapper $textAlignment='start'>
+          <Text size='XL'>Collateral</Text>
+          <CardContainer>
+            <CardRow>
+              <CardRowHeader>
+                <Text size='M' weight='bold'>
+                  Active
+                </Text>
+              </CardRowHeader>
+              <div className='flex flex-col'>
+                {marginAccounts &&
+                  marginAccounts.map((account) => {
+                    const hasAssetsForToken0 = account.assets.token0Raw > 0;
+                    const hasAssetsForToken1 = account.assets.token1Raw > 0;
+                    const hasLiabilitiesForToken0 = account.liabilities.amount0 > 0;
+                    const hasLiabilitiesForToken1 = account.liabilities.amount1 > 0;
+                    const AvailableContainerToken0 =
+                      hasAssetsForToken0 && !hasLiabilitiesForToken1
+                        ? AvailableContainer
+                        : AvailableContainerConnectedLeft;
+                    const AvailableContainerToken1 =
+                      hasAssetsForToken1 && !hasLiabilitiesForToken0
+                        ? AvailableContainer
+                        : AvailableContainerConnectedRight;
+                    const fact = 1 + 1 / ALOE_II_MAX_LEVERAGE + 1 / ALOE_II_LIQUIDATION_INCENTIVE;
+                    let ltv = 1 / (Math.exp((account.nSigma * account.iv) / 10) * fact);
+                    ltv = Math.max(0.1, Math.min(ltv, 0.9)) * 100;
+                    const token0Color = tokenColors.get(account.token0.address);
+                    const token0Gradient = token0Color
+                      ? `linear-gradient(90deg, ${rgba(token0Color, 0.25)} 0%, ${GREY_700} 100%)`
+                      : undefined;
+                    const token1Color = tokenColors.get(account.token1.address);
+                    const token1Gradient = token1Color
+                      ? `linear-gradient(90deg, ${rgba(token1Color, 0.25)} 0%, ${GREY_700} 100%)`
+                      : undefined;
+                    return (
+                      <Fragment key={account.uniswapPool}>
```

Comment:
I don't think this key is sufficient. Would prefer it be a BorrowerNFT ID

Reply:
punting as we don't have that info yet